src/player.rs

```diff
@@ -91,9 +92,10 @@ fn setup(
 	);
 	let atlas_handle = atlases.add(texture_atlas);
 
-	commands
-		.spawn((
-			SpriteBundle {
+	commands.spawn((
+		Player {},
```

Comment:
[Marker component](https://bevy-cheatbook.github.io/programming/ec.html#marker-components) to identify the `Player` entity while querying